packages/nova/DOCUMENTATION.md

```diff
@@ -259,6 +259,22 @@ addLinks(Users, {
 });
 ```
 
+### Foreign keys
+
+By default, Nova uses the field `_id` of the collection B as the foreign field. Sometimes you may want another field to be your foreign key.
+
+You can decide which foreign field to use by specifying `foreignField` in the relationship configuration.
+
+```typescript
```

Comment:
In this scenario we would use Comments (direct link via userId) to do this. Let's give a real world example like the stripe thingie because it helps the user formulate a mental model faster.

Also how would this work with the 'inversed side' we need to add something about that too.

Reply:
Okay, I'll use the stripe example.

And the 'inversed side' has no changes from nominal usage. So do you think it is really necessary to write "to use foreign fields from the inversed side, change nothing" ? 😅

Reply:
yes, it is reasonable to have the inverse side as well.